blueprintapi/settings.py

```diff
@@ -22,11 +23,21 @@
     "handlers": {
         "console": {
             "class": "logging.StreamHandler",
+            "stream": sys.stdout,
+            "formatter": "simple",
         },
     },
-    "root": {
-        "handlers": ["console"],
-        "level": "WARNING",
+    "formatters": {
+        "simple": {
+            "format": "[{asctime}] [{levelname}] {message}",
+            "style": "{",
+        },
+    },
+    "loggers": {
+        "root": {
+            "level": "INFO",
+            "handlers": ["console"],
+        }
```

Comment:
Sorry, this would apply to the logger called "root", but I'm not sure if that actually means that it will work for the root logger. Does it work as expected for all logs? If so, we can just clean this up later.

Reply:
Ah, gotcha. Originally leaving the key in loggers blank came from here -> https://docs.djangoproject.com/en/4.1/howto/logging/#configure-a-logger-mapping

Reply:
Oh, I see! I hadn't seen that before. Nice